samples/core/tutorials/estimators/linear.ipynb

```diff
@@ -399,7 +417,7 @@
       },
       "cell_type": "markdown",
       "source": [
-        "[pandas](https://pandas.pydata.org/) provides some convenient utilities for data analysis. Here's a list of columns available in the Census Income dataset:"
+        "[Pandas](https://pandas.pydata.org/) provides some convenient utilities for data analysis. Here's a list of columns available in the Census Income dataset:"
```

Comment:
Its name is actually "pandas" (lowercase).

Reply:
!

Done.